src/main/java/com/analyzer/App.java

```diff
@@ -1,32 +1,21 @@
 package com.analyzer;
 
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStreamReader;
+import java.util.Map;
 
+public class App {
+    public static void main(String[] args) {
+        // TODO: Scanner which gets all yaml files the config.yaml file --> put it in utils
+        String yamlFilePath = "./activity_definitions/configuration.yaml";
```

Comment:
A path should be in a variable (ENV or start parameter) to be changed by calling the app

Reply:
This is in #7 

I will implement this when I am doing the Scanner class